typedb-src/modules/ROOT/pages/overview.adoc

```diff
@@ -30,28 +30,28 @@ This is the TypeDB server documentation section. It has the following sections:
  ** xref:admin/configuration.adoc[] -- TypeDB server configuration parameters and host machine requirements
  ** xref:admin/export-import.adoc[] -- exporting and importing database schema and data
  ** xref:admin/update.adoc[] -- how to migrate to a newer version of TypeDB
- ** xref:admin/ha.adoc[] -- description of TypeDB high availability features
- ** xref:admin/security.adoc[] -- description of TypeDB security features
+ ** xref:admin/ha.adoc[] -- TypeDB's high availability features
+ ** xref:admin/security.adoc[] -- TypeDB's security features
 * *Tutorials*
  ** xref:tutorials/iam-schema.adoc[] -- a brief description of IAM schema that is widely used throughout the documentation
- ** xref:tutorials/sample-app.adoc[] -- a simple example of how to use TypeDB with Java, Python, and Node.js
+ ** xref:tutorials/sample-app.adoc[] -- a simple example of how to use TypeDB with Java, Python and Node.js
```

Comment:
```suggestion
 ** xref:tutorials/sample-app.adoc[] -- a simple example of how to use TypeDB with Java, Python, and Node.js
```
This is the Oxford comma, we are using it throughout all our documentation.

Reply:
Yes, discussed in Discord.